docs/install/_rpi.rst

```diff
@@ -1,52 +1,73 @@
 
-Kolibri works well on Debian-based distributions for `Raspberry Pi <https://www.raspberrypi.org/>`_, such as Raspbian, and has been tested on RPi 3 models.
 
 Install
 -------
 
-1. Running ``add-apt-repository`` as shown in the PPA instructions does not work. Instead, run::
+#. We need to upgrade the ``python3-cffi`` library, which is outdated on Raspbian. Upgrade it like this::
+
+      sudo apt install libffi-dev
+      sudo pip3 install pip --upgrade
+      sudo pip3 install cffi --upgrade
+      sudo systemctl start kolibri
+
+  .. warning:: If you are installing Kolibri on the older **Raspbian Jessie** which is from before August 2017, you need to complete a few extra steps:
+  
+     ::
+    
+        sudo apt install python3-pip python3-pkg-resources
+        sudo pip3 install setuptools --upgrade
+
+#. Add our Ubuntu PPA with these special instructions::
 
       sudo apt install dirmngr
       sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
       sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys DC5BAA93F9E4AE4F0411F97C74F88ADB3194DD81
       sudo apt update
       sudo apt install kolibri
 
-2. Kolibri does not start after installation. This is because ``python3-cffi`` is outdated on Raspbian. Upgrade it like this::
-
-      sudo apt install libffi-dev
-      sudo pip3 install pip --upgrade
-      sudo pip3 install cffi --upgrade
-      sudo systemctl start kolibri
-
-3. When the command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
+#. When the command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
 
 
 .. note:: The following issues are quite common on a Raspberry Pi:
 
-  * The time isn't set properly and you will have errors downloading software. For instance, SSL certificates for online sources will fail to validate. Ensure that you have the right timezone in ``/etc/timezone`` and that the clock is set properly by running ``sudo ntpd -gq``.
+  * **System time** isn't set properly or resets during power-off. This causes errors while downloading software. For instance, SSL certificates for online sources will fail to validate. Ensure that you have the right timezone in ``/etc/timezone`` and that the clock is set properly by running ``sudo ntpd -gq``.
 
-  * You run out of storage space. If you have a USB source for additional storage, do something like this::
+  * **Storage space** is often a scarcety. If you have a USB source for additional storage, you can use a ``kolibri`` management command or create your own symbolic links to have the data folder located elsewhere.
```

Comment:
> **Storage space** is often scarce.

Can we remove the reference to creating symbolic links? I understand that people might do this and that's OK, but I think of any reason for us to officially support it.

Reply:
I think the recent episode was that the user didn't understand the python command.

I don't believe it's appropriate for us to show alternate methods of doing things that we have an official solution for, unless there are tradeoffs to each method that users need to be aware of.

What's the benefit?

Reply:
Let's say I have this layout:

* SD Card: OS w/ kolibri installation and `~/.kolibri` (containing per-device user database, device registration etc)
* USB attached storage: I keep copying this media, it's bundled with content. I want to reuse them.

I've opened up #52, I will move these more detailed instructions and tips over there later on. I don't see much benefit in discussing the relevance of these detailed tips/instructions for the sake of maintaining simplicity... if the argument is that a detail is too nitty for the general level, then we'll never get to a point where the documentation (field manual) contains details.

I agree that the installation instructions should be simple -- I think we'll get there once the structures (such as a howto) are setup to move these couple of things over.

Would that work?

Reply:
Anyways, I moved it to the HOWTO now to ensure that this PR is ready :+1: